draft-ietf-quic-tls.md

```diff
@@ -1276,10 +1276,31 @@ If 0-RTT keys are available (see {{enable-0rtt}}), the lack of replay protection
 means that restrictions on their use are necessary to avoid replay attacks on
 the protocol.
 
-A client MUST only use 0-RTT keys to protect data that is idempotent.  A client
-MAY wish to apply additional restrictions on what data it sends prior to the
-completion of the TLS handshake.  A client otherwise treats 0-RTT keys as
-equivalent to 1-RTT keys, except that it MUST NOT send ACKs with 0-RTT keys.
+Of the frames defined in {{QUIC-TRANSPORT}}, the STREAM, RESET_STREAM, and
+CONNECTION_CLOSE frames are potentially unsafe for use with 0-RTT as they can
```

Comment:
```suggestion
CONNECTION_CLOSE frames are potentially unsafe for use with 0-RTT as they
```

The three frames are defined to carry application data, I think.

Reply:
I was thinking about the transport-layer CONNECTION_CLOSE.  Maybe that doesn't matter.